src/main/java/org/opensrp/queue/ResourceEvaluatorMessage.java

```diff
@@ -0,0 +1,48 @@
+package org.opensrp.queue;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.ibm.fhir.model.resource.QuestionnaireResponse;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+import lombok.ToString;
+import org.smartregister.domain.Action;
+import org.smartregister.pathevaluator.TriggerType;
+
+import java.io.Serializable;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@ToString
+public class ResourceEvaluatorMessage implements Serializable {
+
+	private static final long serialVersionUID = -1607322794388098644L;
+
+	@JsonProperty
+	private String resource;
+
+	@JsonProperty
+	private QuestionnaireResponse questionnaireResponse;
+
+	@JsonProperty
+	private Action action;
+
+	@JsonProperty
+	private String planIdentifier;
+
+	@JsonProperty
+	private String jurisdictionCode;
+
+	@JsonProperty
+	private TriggerType triggerType;
+
+	@Override
+	public String toString() {
+		String result = "QuestionnareResponse id is: null " + "Action Identifier is : " + getAction() != null ? getAction().getIdentifier() : "null" + "PlanIdentifier : " + getPlanIdentifier() + ", Jurisdiction Code : " + getJurisdictionCode() + ", TriggerType : " + getTriggerType();
```

Comment:
`QuestionnareResponse id is: null` is that a typo?

Reply:
QuestionarreREsponse is always passed as null from the plan evaluator library.